drivers/i3c/i3c_common.c

```diff
@@ -1380,6 +1391,15 @@ int i3c_bus_init(const struct device *dev, const struct i3c_dev_list *dev_list)
 		LOG_DBG("Broadcast RSTDAA was NACK.");
 	}
 
+	/* Set OD timings for Normal speed */
+	if (api->set_speed != NULL) {
+		ret = api->set_speed(dev, I3C_OPEN_DRAIN_NORMAL_SPEED);
+		if (ret != 0) {
+			LOG_ERR("Open Drain Normal speed set failed");
+			return ret;
+		}
+	}
+
```

Comment:
can you separate these changes in to a separate commit (which will use the `i3c_configure` api from the other comment).